cmd/cmd_test.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+)
+
+const cfgHomeKey = "EXERCISM_CONFIG_HOME"
+
+type CommandTest struct {
+	App            *cobra.Command
+	Cmd            *cobra.Command
+	InitFn         func()
+	TmpDir         string
+	Args           []string
+	OriginalValues struct {
+		ConfigHome string
+		Args       []string
+	}
+}
+
```

Comment:
Is it worth adding this to a doc string?
```
// CommandTest ...
//
// To initialize  it needs 3 of the fields: 
// Cmd, InitFn, and Args (the faked out os.Args)
// 
// For example:
// cmdTest := &CommandTest{
//			Cmd:    configureCmd,
//			InitFn: initConfigureCmd,
//			Args:   test.args,
//		}
```
It'd be good to have this explanation with the struct for future contributors.